src/rpc/RPCHelpers.cpp

```diff
@@ -458,19 +459,20 @@ toExpandedJson(Backend::TransactionAndMetadata const& blobs)
     auto [txn, meta] = deserializeTxPlusMeta(blobs, blobs.ledgerSequence);
     auto txnJson = toJson(*txn);
     auto metaJson = toJson(*meta);
-    insertDeliveredAmount(metaJson, txn, meta);
+    insertDeliveredAmount(metaJson, txn, meta, blobs.date);
     return {txnJson, metaJson};
 }
 
 bool
 insertDeliveredAmount(
     boost::json::object& metaJson,
     std::shared_ptr<ripple::STTx const> const& txn,
-    std::shared_ptr<ripple::TxMeta const> const& meta)
+    std::shared_ptr<ripple::TxMeta const> const& meta,
+    uint32_t date)
 {
     if (canHaveDeliveredAmount(txn, meta))
     {
-        if (auto amt = getDeliveredAmount(txn, meta, meta->getLgrSeq()))
+        if (auto amt = getDeliveredAmount(txn, meta, meta->getLgrSeq(), date))
```

Comment:
Can we do a cpp17 style if statement here?

Reply:
Like : `if (auto amt = getDeliveredAmount(txn, meta, meta->getLgrSeq(), date); amt)`?

Reply:
Yes. I like that better, but if you hate it feel free to leave it